benchmark/buffers/buffer-tojson.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  n: [1e4],
+  len: [0, 10, 256, 4 * 1024]
+});
+
+function main(conf) {
+  var n = +conf.n;
```

Comment:
Personally I'd rather just stick to `var` in benchmark code for the forseeable future. Having issues with `let` and `const` in node core code has left a bit of a "bad taste." While it might seem moot in benchmark code, I'd rather not have to worry about deopts making my benchmarks run unnecessarily longer.